### ISSUE TITLE
Can't set mutable content on bootstrapped DHT

### ISSUE THREAD

**Issue by pldubouilh** [feross/bittorrent-dht#99]
I can't seems to be able to set mutable content on a live session anymore. For instance, this code that was working fine on 5.0.0, is not working anymore - `ready` is not fired and `console.log("\n  DHT reached")` is never reached.

I'll try to pinpoint the problematic version in the next couple of days.

```
var DHT = require('bittorrent-dht')
var ed = require('ed25519-supercop')

[...]

// Increment token
token++

// Set opts
opts = {
  k: pub,
  seq: token,
  v: value,
  sign: function (buf) {
    var sig = ed.sign(buf, pub, priv)
    return sig
  }
}


dht = new DHT({ bootstrap: true, verify: ed.verify })

dht.on('ready', function () {
  console.log("\n  DHT reached")

  // Put stuff
  dht.put(opts, function (errors, hash) {
    console.log("\n  DHT updated")
  })
})
```

**Comment by mafintosh** [comment]
`bootstrap: true` is the problem here it seems as that currently disables the bootstrap nodes because of an bug. I'm gonna push a fix for this. If you don't want to wait simply omitting the bootstrap option should work.

**Comment by mafintosh** [comment]
Should be fixed in 6.3.1
